modules/components/pages/outputs/kafka.adoc

```diff
@@ -10,7 +10,7 @@
 component_type_dropdown::[]
 
 
-The kafka output type writes a batch of messages to Kafka brokers and waits for acknowledgement before propagating it back to the input.
+The `kafka` output writes a batch of messages to Kafka brokers and waits for acknowledgement before propagating them back to the input.
```

Comment:
Does this description mean that the acknowledgement is written back to the input or the messages are?